### PR TITLE
Deprecate PycodestyleBeer.py

### PR DESCRIPTION
**Pull request by andreiraduta1101**
Created a new file PyCodeStyleBear.py to deprecate PycodestyleBear.py

Closes: https://github.com/coala/coala-bears/issues/1152

### CODE REVIEW

File: bears/python/PyCodeStyleBear.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            arguments.append('--ignore=' + pycodestyle_ignore)
+
+        if pycodestyle_select:
+            arguments.append('--select=' + pycodestyle_select)
+
+        arguments.append('--max-line-length=' + str(max_line_length))
+
+        arguments.append(filename)
+
+        return arguments
```
Comment (Mixih) [inline bears/python/PyCodeStyleBear.py]: You  need a newline at the end of the file to comply with PEP8

**Comment by Mixih** [comment]
Also check all coala repos to see if they use this bear. Open an issue to change them to the undepreciated bear if any do.

File: bears/python/PyCodeStyleBear.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+        if pycodestyle_select:
+            arguments.append('--select=' + pycodestyle_select)
+
+        arguments.append('--max-line-length=' + str(max_line_length))
+
+        arguments.append(filename)
+
+        return arguments 
+
```
Comment (Mixih) [inline bears/python/PyCodeStyleBear.py]: No tabs or indentation in the EOF newline please

**Comment by andreiraduta1101** [comment]
@sils

**Comment by sils** [comment]
see issue

**Pull request closed without merging**